app/marketing/views.py

```diff
@@ -885,6 +886,25 @@ def trending_quests():
         ).order_by('-recent_attempts').all()[0:10]
     return quests
 
+def quest_of_the_day():
+    quest = trending_quests()[0]
```

Comment:
is this guaranteed to be different day over day? id be annoyed if i got the same quest every day

Reply:
Well, `trending_quests()` was actually written and approved in #6080. `quest_of_the_day()` only pulls the first of its results

Reply:
And trending_quests() actually filters the quests by **number of recent attempts** and **creation date**.

Wouldn't **number of recent attempts** make the trending quests different each day?

Reply:
i think using the upstream changes is fine.

Reply:
Will rebase on upstream master and fix merge conflicts.

Reply:
Done.